.prettierrc.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
   tabWidth: 2,
   semi: true,
   singleQuote: true,
+  arrowParens: 'always',
```

Comment:
Is there a reason why you added this?

I'm not against it, just wondering if this came out of a discussion. Might lead to some unrelated lines getting checked in in future PRs.

Reply:
this was driving me a bit crazy in the developer site, where eslint was recommending this change, but my file wouldnt autosave with parens, so i added this there.

i dont know if im just not using the right tools / setup, or if we would care about the same concerns for this repo (since it doesnt have the same commit check). but i figured i would add it here also 🙈

Reply:
im cool with removing this though.

Reply:
I just wanted to make sure that it was an intentional adjustment rather than a fix for your local setup. As long as future PRs wont have a ton of noise from this stuff getting updated, then I'm good with it 👍